.github/workflows/deployment.yml

```diff
@@ -1,5 +1,6 @@
 name: Preview Deployment
 on:
+  workflow_dispatch:
```

Comment:
I've added this so we have a manual way to trigger the deployment workflow from branches that are opened against other feature branches as opposed to `develop`.

Reply:
🙏🏼 for this QoL upgrade. I've run into this hurdle a few times in the past.